test/openAPI/features/support/list_methods.js

```diff
@@ -17,24 +27,73 @@ Given(
 );
 
 When(
-  'The request to retrieve the list of REST services and endpoints for a service provider is sent',
-  () =>
-    specListMethods.get(baseUrl).withPathParams({
-      GovStackInstance: 'string',
-      memberClass: 'string',
-      memberCode: 'string',
-      applicationCode: 'string',
-    })
+  'User sends GET request with given {string} as serviceId, required path params and header',
+  serviceId =>
+    specListMethods
+      .get(baseUrl)
+      .withHeaders(acceptHeader.key, acceptHeader.value)
+      .withQueryParams({
+        serviceId: serviceId,
+      })
+      .withPathParams({
+        GovStackInstance: 'string',
+        memberClass: 'string',
+        memberCode: 'string',
+        applicationCode: 'string',
+      })
+);
+
+Then('User receives a response from the listMethods endpoint', async () => {
+  await specListMethods.toss();
+});
+
+Then(
+  'The listMethods endpoint response should be returned in a timely manner',
+  () => {
+    specListMethods
+      .response()
+      .to.have.responseTimeLessThan(defaultExpectedResponseTime);
+  }
 );
 
+Then('The listMethods endpoint response should have status 200', () => {
+  specListMethods.response().to.have.status(200);
+});
+
 Then(
-  'The operation returns the list of REST services and endpoints for a service provider',
-  async () => {
-    specListMethods.expectStatus(200).expectJsonSchema(responseSchema);
-    await specListMethods.toss();
+  'The listMethods endpoint response should have content-type header',
+  () => {
+    specListMethods
+      .response()
+      .should.have.header('content-type', acceptHeader.value);
```

Comment:
```suggestion
      .should.have.header(acceptHeader.key, acceptHeader.value);
```
After change in the helpers file